src/System.Management.Automation/help/UpdatableHelpInfo.cs

```diff
@@ -37,6 +39,44 @@ internal CultureSpecificUpdatableHelp(CultureInfo culture, Version version)
         /// Supported culture.
         /// </summary>
         internal CultureInfo Culture { get; set; }
+
+        /// <summary>
+        /// Enumerates fallback chain (parents) of the culture, including itself.
+        /// </summary>
+        /// <param name="culture">Culture to enumerate</param>
+        /// <example>
+        /// Examples:
+        /// en-GB => { en-GB, en }
+        /// zh-Hans-CN => { zh-Hans-CN, zh-Hans, zh }.
+        /// </example>
+        /// <returns>An enumerable list of culture names.</returns>
+        internal static IEnumerable<string> GetCultureFallbackChain(CultureInfo culture)
```

Comment:
One of the issues is that on Ubuntu 20+, the default culture is set as C.UTF-8. 
It doesn't have 'en' in its parent chain. Can we accommodate for cases like that? If a matching culture is not found fallback to en-US?

Reply:
@adityapatwardhan Interesting case, but out of scope. From my skimming of the POSIX standard, `C.UTF-8` is more of a default null culture and in no way equivalent to `en-US`. In fact, presuming everyone uses `en-US` is the very source of the issues I am attempting to fix.

Source issue shows the error that will be shown, I think it is well good enough:

```
Update-Help: Failed to update Help for the module(s) 'Microsoft.PowerShell.Core' with UI culture(s) {C}:
Unable to retrieve the HelpInfo XML file for UI culture C.
Make sure the HelpInfoUri property in the module manifest is valid or check your network connection and then try the command again..
English-US help content is available and can be installed using: Update-Help -UICulture en-US.
```

I could specialise the error message to also show something like "Your culture is C.UTF-8, which is a default not associated with any language, consider changing your system culture", but please create another issue for it so that it can be discussed.

Reply:
I think the updated error message is much better than the current one. It would be great if you could do that.

Reply:
Here: https://github.com/PowerShell/PowerShell/issues/19633